pages/mozfest/index.vue

```diff
@@ -1,5 +1,9 @@
 <template>
   <div>
+    <social-head
+     title="MozFest - Here to learn"
+     description=""
```

Comment:
leaving this empty results in empty twitter and og descriptions. Perhaps use a default value in SocialHead for the title and description props?

```
<meta data-n-head="ssr" data-hid="twitter:description" name="twitter:description" content="">
<meta data-n-head="ssr" data-hid="og:description" property="og:description" content="">
```
view-source:https://gradual-qpbwuzrfd-phivk.vercel.app/mozfest/

similar for other uses of SocialHead component